docs/development/issue-labeling.md

```diff
@@ -34,14 +34,13 @@ Most issues should have a label relating to either a platform, manager, datasour
 
     status:requirements
     status:blocked
-    status:ready
     status:in-progress
 
 </details>
 
 Use these to label the status of an issue.
 For example, use `status:requirements` to mean that an issue is not yet ready for development to begin.
-All open issues should have some `status:*` label applied, and [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Astatus%3Arequirements+-label%3Astatus%3Aready+-label%3Astatus%3Ain-progress+-label%3Astatus%3Ablocked+-label%3Astatus%3Awaiting-on-response+) can find any which are missing a status label.
+All open issues should have some `status:*` label applied, and [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Astatus%3Arequirements+-label%3Astatus%3Ain-progress+-label%3Astatus%3Ablocked) can find any which are missing a status label.
```

Comment:
```suggestion
```

We will have no status label from now on, instead of status:ready

Reply:
Do you _really_ mean to remove `status:requirements`, `status:blocked` and `status:in-progress`?

I thought you wanted to keep the `status:requirements` escape-hatch to convert discussion that you want to track? Or do you want to have a new label for that concept?

And what about `status:blocked` to track where we need internal/external things done first?

I don't know if `status:in-progress` is valuable to you?

Reply:
No, I mean to remove the sentence, which says "all open issues should have some status: label applied" - this will no longer be true